### ISSUE TITLE
Reduced CSF copper concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9850]
**Preferred term label:**
Reduced CSF copper concentration

**Synonyms**

**Definition (free text, please give PubMed ID)**
Concentration of copper in the cerebrospinal fluid below the lower limit of normal.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
620306 
PMID:35913762  2/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal
added as
Decreased CSF copper concentration
HP:0034823
